letsencrypt/tests/cli_test.py

```diff
@@ -117,7 +127,7 @@ def test_auth_bad_args(self):
         self.assertEqual(ret, '--domains and --csr are mutually exclusive')
 
         ret, _, _, _ = self._call(['-a', 'bad_auth', 'auth'])
-        self.assertEqual(ret, 'Authenticator could not be determined')
+        self.assertEqual(ret, 'The requested bad_auth plugin does not appear to be installed')
```

Comment:
This line is long, please break.

Also, use consistent type of quotes throughout a single file: either `"` OR `'`. `cli_test` is already messed up anyway...